### ISSUE TITLE
"green threads"

### ISSUE THREAD

**Issue by carols10cents** [chriskrycho/async-trpl-fun#1]
Re: green threads--

There used to be a discussion of green threads in the book, but [apparently we messed it up](https://github.com/rust-lang/book/issues/2497), so [I removed it](https://github.com/rust-lang/book/pull/3027/commits/2b1dc38e4ea59585a36e294900b49936486bcd7d). 🤷🏻‍♀️ 

If it makes sense to mention it, I'm not opposed to putting it back (with an actually correct definition, of course!)

**Comment by chriskrycho** [comment]
Yes! Notably several of the runtimes use the term in their docs but do not define it. 😂
